Tests/PostgresNIOTests/New/Connection State Machine/ConnectionStateMachineTests.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 06.01.21.
+//
+
+import XCTest
```

Comment:
Ditch Xcode template headers.
```suggestion
import XCTest
```